src/main/java/com/google/sps/models/AppSeeder.java

```diff
@@ -0,0 +1,28 @@
+package com.google.sps.servlets;
```

Comment:
this isn't a model. can you move this to google/sps/seeder and update the package name accordingly as well?